website/docs/d/sensitive_file.html.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "local"
+page_title: "Local: local_sensitive_file"
+sidebar_current: "docs-local-datasource-sensitive_file"
```

Comment:
Not for this pull request, but `sidebar_current` is no longer necessary in Terraform documentation.